doc/Development_Documentation/05_Objects/01_Object_Classes/05_Class_Settings/26_Interfaces_and_traits.md

```diff
@@ -0,0 +1,74 @@
+# Using interfaces and traits
+In some cases you need to implement interfaces and overwrite some provided functions.
+
+##### Example
+I used the demo project to show the usage of this feature.
+Extending cars to have a gearbox function to indidcate which type of shifting is used. This will be done with an interface and a trait to implement the functions.
```

Comment:
```suggestion
We're extending the `Cars` class with methods to retrieve the transmission type as well as the amount of gears. This will be done by implementing an interface and a trait that adds the required methods.
```